tests/assertions.js

```diff
@@ -77,7 +77,10 @@ export const verificationSuccess = async ({credential, verifier}) => {
       checks: ['proof']
     }
   };
-  const {result, error} = await verifier.post({json: body});
+  const results = await verifier.post({json: body});
+
+  const {result, error} = results;
```

Comment:
why not just stick with the original destructuring?

Reply:
Yeah I can put that back - this is a side effect of changes I made while debugging

Reply:
yeah I had a feeling you logged results to see what's inside of it. code for that is partially here: https://github.com/w3c/vc-test-suite-implementations/blob/d038e90dd7581c435b7dbada3e3a43d7aa4886c9/lib/requests.js#L60

There's a second one for zcap calls after it.